docs/GettingStarted.md

```diff
@@ -3,47 +3,252 @@
 ## Pre-requisites
 
 - An Azure Subscription with the necessary permissions to create resources and application registrations.
-- A Microsoft Dataverse Environment.
-- A Dynamics 365 installation with Field Service.
 - A Linux or WSL2 Environment.
+- A GitHub client.
 - Visual Studio Code.
+- (Optional) A Microsoft Dataverse Environment.
+- (Optional) A Dynamics 365 installation with Field Service.
 
-## Preparation
 
-- Install the Windows Subsystem for Linux (WSL2) in your Windows computer or a suitable Linux Distribution
-- Install dotnet-sdk-6.0
-- Install Visual Studio Code
+## Deployment Steps
+
+In the following deployment steps, you will:
+
+- Install all the necessary pre-requisites
+- Deploy the telemetry platform layer, the fleet integration layer and the required azure functions
+- Test connectivity and visualize results
+- Clean-up resources
+
+### Preparation
+
+- If you don't have an Azure subscription, you can [create a free account](https://azure.microsoft.com/en-us/pricing/purchase-options/azure-account).
+
+- Install the Windows Subsystem for Linux (WSL2) in your Windows computer or use a suitable Linux Distribution such as Ubuntu 22.04 LTS.
+
+- Install dotnet-sdk-8.0. Read [Install .NET on Linux](https://learn.microsoft.com/en-us/dotnet/core/install/linux) and follow the steps for your distro.
+
+- Install [Visual Studio Code](https://visualstudio.microsoft.com/)
+
 - Make sure you have [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt) installed. Run `az --version` to verify. If it's not installed, run the following command to install it:
 
 ```bash
 curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
 ```
 
+- Install the [Azure Functions core tools](https://github.com/Azure/azure-functions-core-tools/blob/v4.x/README.md#linux) to work with Azure functions.
 - Check out the GitHub repository in your environment
 
 ``` bash
 git clone https://github.com/microsoft/connected-fleet-refarch.git
 ```
 
-- Open Visual Studio Code from WSL using the argument
+- If using WSL, Open Visual Studio Code from WSL on the code directory
 
 ``` bash
 cd connected-fleet-refarch
 code .
 ```
 
-- Install the following extensions in Visual Studio Code
+- If you want to modify the function code, install the following extensions in Visual Studio Code to make your life easier.
   - [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep): Bicep language support for Visual Studio Code
   - [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions): quickly create, debug, manage, and deploy serverless apps directly from VS Code
 
-## Deployment
+### Deploying the telemetry platform layer
```

Comment:
Can we have numbered steps to follow? It helps learners to keep track of where they are.